### ISSUE TITLE
Please add BlackArch distro mirrors

### ISSUE THREAD

**Issue by karmanyaahm** [RubenKelevra/pacman.store#61]
Hello, 

It's really nice to have these IPFS mirrors for regular Arch Linux. 

However, I still pull packages for https://www.blackarch.org from HTTP mirrors. The blackarch repos provide many tools (https://www.blackarch.org/tools.html) in addition to regular arch.
It would be really cool if you could make an IPFS mirror for that as well.

https://www.blackarch.org/downloads.html#mirror-list

**Comment by RubenKelevra** [comment]
Hey @karmanyaahm,

thanks for the recommendation.

I'm currently not actively adding stuff on my own, just on requests. So you did the right thing :)

It depends a bit how the sync is beeing done, as I really don't want to implement another way to check for changes, as I think manjaro (?) is still not properly working and often doing "null" changes.

If its easy and still fitting the hard drive space on the import server I'll let you know :)

**Comment by RubenKelevra** [comment]
Currently the whole operation is sadly blocked by some ipfs bugs:

https://github.com/RubenKelevra/pacman.store/issues/62

Afterwards I give this a shot :)

**Comment by RubenKelevra** [comment]
@karmanyaahm If you're interested, I'm happy to help you hosting this yourself on a collaborative cluster. :)

**Comment by karmanyaahm** [comment]
> @karmanyaahm If you're interested, I'm happy to help you hosting this yourself on a collaborative cluster. :)

I would love to but my only server that has enough storage, my NAS, has a very mediocre CPU where running IPFS wouldn't be practical 🙁. If I have the means in the future, I'll try to set up a cluster.
